kolibri/plugins/learn/assets/src/views/LearnIndex.vue

```diff
@@ -14,8 +14,8 @@
 
   import { mapGetters, mapState } from 'vuex';
   import NotificationsRoot from 'kolibri.coreVue.components.NotificationsRoot';
-  import { PageNames } from '../constants';
   import plugin_data from 'plugin_data';
+  import { PageNames } from '../constants';
```

Comment:
is this just from linting?

Reply:
Yes, for reasons I couldn't discern, linting changed its mind while I was making this PR and decided to move things around.